spec/rubocop/formatter/clang_style_formatter_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe RuboCop::Formatter::ClangStyleFormatter do
+RSpec.describe RuboCop::Formatter::ClangStyleFormatter, :cop do
```

Comment:
Maybe we can have a separate context for formatters?

Reply:
We could, I haven't checked if there was extra things that could be factorized.

Reply:
Didn't see much of interest